page.php

```diff
@@ -21,18 +21,35 @@
  * @since    Timber 0.1
  */
 
+$copyrightText = get_option( 'copyright' );
```

Comment:
Use this closer to setting up var that's re-used. So, I'd remove this line and on line 50 I'd add
`$context['copyright_text']        = get_option( 'copyright', '' ) ? get_option( 'copyright' ) : '';`

Reply:
Done